eth/watchers/timewatcher.go

```diff
@@ -122,6 +124,19 @@ func (tw *TimeWatcher) Watch() error {
 		return fmt.Errorf("error fetching initial transcoderPoolSize err=%v", err)
 	}
 
+	b, err := tw.lpEth.Backend()
+	if err != nil {
+		return err
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 8*time.Second)
+	defer cancel()
+	lastSeenBlock, err := b.HeaderByNumber(ctx, nil)
```

Comment:
Instead of fetching the latest block number via RPC request here could we use the latest block number returned by `tw.watcher.GetLatestBlock()`? If this value is behind the latest block height, then `tw.lastSeenBlock` will be initialized to this value and then it will be updated when backfilling is complete. If the block watcher has no latest block then `tw.lastSeenBlock` will be nil at first, but it will also be updated when backfilling is complete.

Reply:
This issue occurs when we do not backfill so after backfilling is complete `tw.lastSeenBlock` will still be `nil` and we'll still run into a nil-pointer error when receiving a `GetOrchestratorInfo` request before the first block is propagated.

Reply:
I believe that backfilling does not occur when the ETH node's latest block is less than or equal to the block watcher's latest block. Wouldn't this imply that the block watcher has a latest block and that `tw.lastSeenBlock` is initialized to a non-nil value? The only time `tw.lastSeenBlock` is initialized to nil would be if the block watcher does not have a latest block, but that would imply that backfilling will occur and set `tw.lastSeenBlock` to a non-nil value.

Reply:
Oh yes I see what you're saying. Sorry I read your initial comment a little too fast.

Reply:
fixed in a53f76fd

I commented out the stubBackend I wrote but can remove it if you want.